controllers/persistent-volume-claim/controller.go

```diff
@@ -92,9 +92,7 @@ func (r *PersistentVolumeClaimReconciler) Reconcile(ctx context.Context, req ctr
 	for _, node := range nodeList.Items {
 		capacity, ok := node.Annotations[capacityAnnotation]
 		if !ok {
-			errMessage := fmt.Sprintf("could not find capacity annotation on the node %s", node.Name)
-			logger.Error(fmt.Errorf(errMessage), errMessage)
-			return ctrl.Result{}, nil
+			continue
```

Comment:
Could we write a test case for the controller not erroring?

Reply:
On that same note, couldn't it happen that there is more than one deviceClass present on TopoLVM side? Then this annotation would increment +1, right?

Reply:
I was just thinking about the multiple deviceClasses case where we would actually need to check the capacity label corresponding to the storage class that is specified in the requesting pvc.

Reply:
There are multiple annotations based on the number of deviceClasses. The annotation I'm using here is showing the capacity of the default deviceClass.

Reply:
Im trying to understand if its possible to have a non-default deviceClass be relevant here. Afaik we controll the lvmd config from lvm operator so we should never come into a non-default case right?

Reply:
Topolvm errors if there is no single default deviceClass specified in lvmd config. However, here it is not much relevant. 
- Let's say the user specified two deviceClasses (`vg1` and `vg2`). 
- In this case, lvms creates two storageClasses (`lvms-vg1` and `lvms-vg2`)
- Also, Topolvm adds a capacity label to each node for both deviceClasses (`capacity.topolvm.io/vg1: 96427048960`, `capacity.topolvm.io/vg2: 192979927040`), and one for the default deviceClass which is vg1 in this case (`capacity.topolvm.io/00default: 96427048960`). 

In this implementation of capacity warning, we should check which storageClass is being used in the pvc and print the capacity for that storageClass/deviceClass using the capacity annotations. This is irrelevant to whether deviceClass is the default or not.

Reply:
Then in this case the PVC check here needs to change to accommodate for the correct capacity annotation I believe? Since if we check for default but another deviceClass would be used it would be irrelevant for the PVC?

Reply:
Exactly. I'll update the implementation.